css/style.css

```diff
@@ -507,6 +514,10 @@ body {
     max-width: 373px;
   }
 
+/*  .header__right_row {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5708338)